packages/menu/src/MenuGroup/MenuGroup.types.ts

```diff
@@ -0,0 +1,12 @@
+export interface MenuGroupProps {
+  /**
+   * Content that will appear inside of MenuGroup component.
+   * @type `<MenuItem />` | `<SubMenu />` | `<MenuGroup />` | `<MenuSeparator />`
+   */
+  children: React.ReactNode;
+
+  /**
+   * className that will be applied to root MenuGroup element.
+   */
+  className?: string;
```

Comment:
Shouldn't need `className`, and should extend an HTML element instead

Reply:
👍